Update sso docs (#1012)
harness/developer-hub: docs/first-gen/firstgen-platform/security/access-management-howtos/single-sign-on-sso-with-saml.md

6. Select **Add Entity ID** and enter your custom Entity ID. The default Entity ID is **app.harness.io**. The value you enter here will override the default Entity ID.
7. Click **SUBMIT**. The SAML SSO Provider is now set up to use the Group Attribute Name for authorization.

> 8. Link the SAML SSO Provider to the Harness User Group. We will create a new Harness User Group, but you can use an existing Group so long as your Harness User account is a member and that User account is registered using the same email address you used to register with your SAML provider. Please do not use the default **Account Administrator** group for linking the SSO, as it puts the default group under the risk of deletion.  

Reviewer: ```suggestion
8. Link the SAML SSO Provider to the Harness User Group. We will create a new Harness User Group, but you can use an existing Group so long as your Harness User account is a member and that User account is registered using the same email address you used to register with your SAML provider.

:::important

SSO should not be linked to the default **Account Administrator** group because this puts the group at risk of deletion.
:::

```

Reviewer: To draw attention to this, it would be better to place it within a note.
